workshop/1-Personalization/Lab-4-Evaluate-recommendations.ipynb

```diff
@@ -630,7 +838,7 @@
     "If you don't see the Personalize service icon and recipe name, the page is using default behavior of displaying products from the same category\n",
     "(verify that the campaign was created successfully above **and** the campaign ARN is set as an SSM parameter).\n",
     "\n",
-    "![Related Product Recommendations](./images/retaildemostore-related-products.jpg)"
+    "![Related Product Recommendations](https://raw.githubusercontent.com/aws-samples/retail-demo-store/master/workshop/1-Personalization/images/retaildemostore-related-products.jpg)"
```

Comment:
Link contains the branch "master".

Reply:
Same as above. Images were broken throughout the notebook. I couldn't figure out how to keep relative links working.

Reply:
Now relative links are working again. Don't know why they were breaking before. Reverted to relative.